### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by rbehjati**
# Checklist

- [x] Pull request only updates documentation 
- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by [Cloudbuild](cloudbuild.yaml)
  - [ ] I have updated [documentation](docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -11,18 +11,25 @@ Inside Docker, we use Bazel to version, install and build dependencies and our
...
     with)
-  - add `source $HOME/.cargo/env` to your shell init script (e.g. `.bashrc` or
+    - add `source $HOME/.cargo/env` to your shell init script (e.g. `.bashrc` or
     `.zshrc`)
-  - `rustup target add wasm32-unknown-unknown`
+  - Add WebAssembly target to be able to compile to WebAssembly 
+  (see [Rust Platform Support](https://forge.rust-lang.org/release/platform-support.html)):
+    - `rustup target add wasm32-unknown-unknown`
+- Protocol Buffers:
+  - Install protobuf-compiler: `apt install protobuf-compiler`
```
Comment (tiziano88) [inline docs/development.md]: I think this installs a random version which may or may not be what we use in CI, but I guess that applies to everything else in here. Maybe mention something about this at the beginning of this list, or in the previous paragraph about Dockerfile (i.e. that only the instructions in the Dockerfile contain the accurate versions used, these manual steps are just an approximation).

**Comment by tiziano88** [comment]
Just checking: is this ready for review again? if so, please re-request review from the GitHub UI, thanks!

**Comment by rbehjati** [comment]
There was a problem with formatting which is now fixed.

File: docs/development.md [hunk 2]
```diff
@@ -11,20 +11,33 @@ Inside Docker, we use Bazel to version, install and build dependencies and our
...
+    - `less /tmp/rustup` (inspect downloaded script before running it)
+    - `sh /tmp/rustup` (follow on-screen instructions -- option 1 is fine to
+      start with)
+    - add `source $HOME/.cargo/env` to your shell init script (e.g. `.bashrc` or
+      `.zshrc`)
+  - Add WebAssembly target to be able to compile to WebAssembly (see
+    [Rust Platform Support](https://forge.rust-lang.org/release/platform-support.html)):
+    - `rustup target add wasm32-unknown-unknown`
+- Protocol Buffers:
+
```
Comment (tiziano88) [inline docs/development.md]: extra line?

File: docs/development.md [hunk 3]
```diff
@@ -11,20 +11,33 @@ Inside Docker, we use Bazel to version, install and build dependencies and our
...
+  - Install protobuf-compiler: `apt install protobuf-compiler`
+
+[Step by step instructions for installing Oak on Ubuntu 18.04](/INSTALL.md)
+shows how to install the prerequisites starting off with a clean Ubuntu install.
+Note the server runs in the Docker container but the examples run on the host
```
Comment (tiziano88) [inline docs/development.md]: Perhaps split the paragraph before this line.

File: docs/development.md (hunk 2 above)
Comment (rbehjati) [reply docs/development.md]: Good catch!

**Pull request merged**